packages/farcaster-js-hub-rest/src/utils.ts

```diff
@@ -11,9 +11,14 @@ export function hexToSigner(signerHex: string): NobleEd25519Signer {
   return new NobleEd25519Signer(privateKeyBytes);
 }
 
-export function eip712SignerFromPhrase(mnemonic: string): Eip712Signer {
-  const wallet = Wallet.fromPhrase(mnemonic);
-  return new EthersEip712Signer(wallet);
+export function eip712SignerFromMnemonicPrPrivateKey(mnemonicPrPrivateKey: string): Eip712Signer {
+  let account;
+  if (mnemonicPrPrivateKey.startsWith('0x')) {
```

Comment:
Big nitpick here, but the detail that the private key must start with `0x` is a bit hidden from the caller of `submitVerification`.

This probably wouldn't matter so much, except for the fact that some wallets (Metamask for example) will export an account's private key without the leading `0x`

Reply:
updated to work with or without 0x prefix